docs/guide/input-forms.md

```diff
@@ -119,26 +123,79 @@ class like it is done in the above example with [[yii\helpers\Html::submitButton
 > }
 > ```
 
-Creating Drop-down List <span id="creating-activeform-dropdownlist"></span>
+Creating Lists <span id="creating-activeform-dropdownlist"></span>
 -----------------------
 
-We can use ActiveForm [dropDownList()](http://www.yiiframework.com/doc-2.0/yii-widgets-activefield.html#dropDownList()-detail)
-method to create a drop-down list:
+There are 3 types of lists:
+* Dropdown lists 
+* Radio lists
+* Checkbox lists
+
+To create a list, you have to prepare the items. This can be done manually:
 
 ```php
-use app\models\ProductCategory;
+$items = [
+    1 => 'item 1', 
+    2 => 'item 2'
+]
+```
+
+or by retrieval from the DB:
+```php
+$items = Category::find()
+        ->select(['id', 'label'])
+        ->indexBy('id')
+        ->column();
+```
 
-/* @var $this yii\web\View */
+These $items have to be processed by the different list widgets.
+The value of the form field (and the current active item) will be automatically set 
+by the current value of the $model's attribute. 
+
+#### Creating a drop-down list <span id="creating-activeform-dropdownlist"></span>
+
+We can use ActiveField [dropDownList()](http://www.yiiframework.com/doc-2.0/yii-widgets-activefield.html#dropDownList()-detail)
+method to create a drop-down list:
+
+```php
 /* @var $form yii\widgets\ActiveForm */
-/* @var $model app\models\Product */
 
-echo $form->field($model, 'product_category')->dropdownList(
-    ProductCategory::find()->select(['category_name', 'id'])->indexBy('id')->column(),
+echo $form->field($model, 'category')->dropdownList([
+        1 => 'item 1', 
+        2 => 'item 2'
+    ],
     ['prompt'=>'Select Category']
 );
 ```
 
-The value of your model field will be automatically pre-selected.
+#### Creating a radio list <span id="creating-activeform-dropdownlist"></span>
+
+We can use ActiveField [radioList()](http://www.yiiframework.com/doc-2.0/yii-widgets-activefield.html#radioList()-detail)
+method to create a radio list:
+
+```php
+/* @var $form yii\widgets\ActiveForm */
+
+echo $form->field($model, 'category')->radioList([
+    1 => 'radio 1', 
+    2 => 'radio 2'
+]);
+```
+
+#### Creating a checkbox List <span id="creating-activeform-dropdownlist"></span>
+
+We can use ActiveField [checkboxList()](http://www.yiiframework.com/doc-2.0/yii-widgets-activefield.html#checkboxList()-detail)
```

Comment:
It's referenced differently. Should be `[[\yii\widgets\ActiveField::checkboxList()]]`.